src/plot_data.jl

```diff
@@ -439,12 +439,12 @@ function categorize_data(
     var_types = Dict(zip(last.(split.(string.(keys(data)), "_")), keys(data)))
     for (category, list) in aggregation
         category_df = DataFrames.DataFrame()
-        for tuple in list
-            if haskey(var_types, tuple[1])
-                category_data = data[var_types[tuple[1]]]
+        for (component_type, variable) in list
+            if haskey(var_types, component_type)
+                category_data = data[var_types[component_type]]
                 colname =
-                    typeof(names(category_data)[1]) == String ? "$(tuple[2])" :
-                    Symbol(tuple[2])
+                    typeof(names(category_data)[1]) == String ? "$variable" :
+                    Symbol(variable)
```

Comment:
Does DataFrames not care if you pass String or Symbol? I see that this is existing code, but should we be consistent?

Reply:
I'm pretty sure that DataFrames does not care.

Reply:
When switched to just `variable`, tests still pass, and it works for me.